src/TesApi.Web/AzureProxy.cs

```diff
@@ -408,12 +408,12 @@ public IAsyncEnumerable<CloudPool> GetActivePoolsAsync(string hostName)
             };
 
             return batchClient.PoolOperations.ListPools(activePoolsFilter).ToAsyncEnumerable()
-                .Where(p => p.Metadata.Any(metadata => metadata.Name switch
+                .Where(p => p.Metadata?.Any(metadata => metadata.Name switch
```

Comment:
do all legacy TES pools have Metadata?  Any concerns about orphaning pools with this change?